### ISSUE TITLE
The "anwser" for some examples in "qasper.jsonl" is strange

### ISSUE THREAD

**Issue by Zcchill** [THUDM/LongBench#67]
I download the data from the offcial url and I found that the "answers" of several examples in "qasper.jsonl" are confusing. Here are several examples:
{"pred": "No", "answers": ["Yes", "No"], "all_classes": null, "length": 2317,  "input": "Does this method help in sentiment classification task improvement?", "_id": "bcfe56efad9715cc714ffd2e523eaa9ad796a453e7da77a6"}
{"pred": "unanswerable", "answers": ["Yes", "Unanswerable"], "all_classes": null, "length": 2284, "actual_length": 3533, "input": "Is jiant compatible with models in any programming language?", "_id": "e5d1d589ddb30f43547012f04b06ac2924a1f4fdcf56daab"}
{"pred": "BERTBase", "answers": ["BERTbase", "BERTbase"], "all_classes": null, "length": 3852, "actual_length": 5701, "input": "What BERT model do they test?", "_id": "2a51c07e65a9214ed2cd3c04303afa205e005f4e1ccb172a"}

**Comment by bys0318** [comment]
This might be due to the model iteration. We test the GPT-3.5-Turbo-16k at August, 2023. I think it has a different version now.

**Comment by bys0318** [comment]
You're right. We want to emphasize the task instruction, so we insert the instruction at both the start and the end of the input.
